### DISCUSSION TITLE
fatal: unable to access '<repo_url>': Failure when receiving data from the peer

### DISCUSSION THREAD

**Discussion by kapral18** [community/community#70049] [Repositories] [answered]
I suddenly started receiving this error today during batch git pulls. And it just keeps happening. I am using neovim as an editor and lazy.nvim as package manager that does package update checks which are github plugins for neovim, and this error basically blocks consistently some portion of those updates each time.

I also tried manually running git pull outside of neovim, and I hit the same error eventually after ~10 subsequent git pulls.

![CleanShot 2023-10-14 at 07 29 32@2x](https://github.com/community/community/assets/1625373/f10f0b09-951c-44a3-8330-21f931baa24d)

![CleanShot 2023-10-14 at 07 27 05@2x](https://github.com/community/community/assets/1625373/708ed6fa-9b57-4c0d-ada3-81e3404024e9)

Afaik github pull rate limit is very high so there is no way I hit it within an hour, so this should be something else...

Does anybody have an idea what this can be?

My specs:

location: Netherlands
station: mb pro m1 max
os: sonoma 14.0
terminal: iterm2 3.5.0.beta15
git: 2.42.0

Thanks.

**Comment by pillmuncher** [comment]
Having the same problem, seemingly starting with the upgrade of Neovim from 0.9.2 to 0.9.4 a few minutes ago. I didn't run `:Lazy update` before the upgrade, so I don't really know if that's the reason.

BTW: When running `:Lazy update` repeatedly, every time a different set of plugins from before isn't fetched, meaning that after a few runs all will have been updated.

**Reply by kapral18** [reply]
I tried also outside of neovim and I face the same issue if I repeatedly call `git pull`. Updated my op with screenshot.

**Comment by neuromaancer** [comment]
Same here, but I tried with an VPN to the states, it works fine but slow. I think it is a GitHub network issue. BTW, I assume your IPs are not from states?

**Reply by kapral18** [reply]
I tried with different VPN servers from and outside US - no difference. I reside in Netherlands. But I think it also takes into account my git credentials and matches with the account and account's location.

**Comment by gagbo** [comment]
Having the same issue currently with Doom Emacs, which also happens to batch git pull operations to update emacs packages. And indeed, I'm in EU, not US (and it's like 1:30 AM in Seattle now, so not expecting too much now...)

**Comment by nickersk** [comment]
openembedded image build same issue ...

**Comment by atvcaptain** [comment]
i have same issue fetch git refs

git 2.42.0
location: germany RZ hetzner

./git-loop.sh
694c74aa6c65ffa8a32847eebc7be072aee185b9        HEAD
694c74aa6c65ffa8a32847eebc7be072aee185b9        refs/heads/master
4cb4a7d2582c6f3b55c7007cbe4941d9fbd26488        refs/import/heads/master
0dc4b4611fb739bc08a324c8c71f68e2af7deafb        refs/import/tags/0.4.0
210122cc65438c4eb60866d05fd53e6da7edddbe        refs/import/tags/0.4.0.RC1
d184e816539a238df702118c9b82c15ccf0df1eb        refs/pull/2/head
0dc4b4611fb739bc08a324c8c71f68e2af7deafb        refs/tags/0.4.0
210122cc65438c4eb60866d05fd53e6da7edddbe        refs/tags/0.4.0.RC1
694c74aa6c65ffa8a32847eebc7be072aee185b9        HEAD
694c74aa6c65ffa8a32847eebc7be072aee185b9        refs/heads/master
4cb4a7d2582c6f3b55c7007cbe4941d9fbd26488        refs/import/heads/master
0dc4b4611fb739bc08a324c8c71f68e2af7deafb        refs/import/tags/0.4.0
210122cc65438c4eb60866d05fd53e6da7edddbe        refs/import/tags/0.4.0.RC1
d184e816539a238df702118c9b82c15ccf0df1eb        refs/pull/2/head
0dc4b4611fb739bc08a324c8c71f68e2af7deafb        refs/tags/0.4.0
210122cc65438c4eb60866d05fd53e6da7edddbe        refs/tags/0.4.0.RC1
694c74aa6c65ffa8a32847eebc7be072aee185b9        HEAD
694c74aa6c65ffa8a32847eebc7be072aee185b9        refs/heads/master
4cb4a7d2582c6f3b55c7007cbe4941d9fbd26488        refs/import/heads/master
0dc4b4611fb739bc08a324c8c71f68e2af7deafb        refs/import/tags/0.4.0
210122cc65438c4eb60866d05fd53e6da7edddbe        refs/import/tags/0.4.0.RC1
d184e816539a238df702118c9b82c15ccf0df1eb        refs/pull/2/head
0dc4b4611fb739bc08a324c8c71f68e2af7deafb        refs/tags/0.4.0
210122cc65438c4eb60866d05fd53e6da7edddbe        refs/tags/0.4.0.RC1
fatal: unable to access 'https://github.com/oe-mirrors/python3-yenc.git/': Empty reply from server

**Comment by jbleyel** [comment]
Please add HIGH and BUG as label.
This needs to be fixed ASAP.

**Reply by kapral18** [reply]
fixed, see answer below

**Comment by gagbo** [comment, marked as answer]
It seems resolved for me (in France), as of just now (had my update script running in a loop). So from my point of view, there was a 15 hour outage in "Git Operations", that just never got detected/acknowledged on https://githubstatus.com

**Reply by atvcaptain** [reply]
yes works again, here 20h the issue

**Reply by nickersk** [reply]
yep, works here too now

**Reply by neuromaancer** [reply]
works again!
